platform/vs/docker-sonic-vs/start.sh

```diff
@@ -5,7 +5,22 @@
 PLATFORM=x86_64-kvm_x86_64-r0
 HWSKU=Force10-S6000
 
-ln -sf /usr/share/sonic/device/$PLATFORM/$HWSKU /usr/share/sonic/hwsku
+if [[ ! -d /usr/share/sonic/hwsku ]]; then
+    ln -sf /usr/share/sonic/device/$PLATFORM/$HWSKU /usr/share/sonic/hwsku
+    SYSTEM_MAC_ADDRESS=$(ip link show eth0 | grep ether | awk '{print $2}')
+    sonic-cfggen -a '{"DEVICE_METADATA":{"localhost": {"mac": "'$SYSTEM_MAC_ADDRESS'"}}}' --print-data > /etc/sonic/init_cfg.json
+elif [[ -f /usr/share/sonic/hwsku/default_config.json ]]; then
+    cp /usr/share/sonic/hwsku/default_config.json /etc/sonic/init_cfg.json
+    while [[ ! -f /sys/class/net/eth32/operstate ]]; do
```

Comment:
How do we inject < 32 ports into the virtual chassis?

Reply:
we will address this in PR2.

Reply:
we will address this in PR2.